web/html/wallet.js

```diff
@@ -62,6 +62,18 @@ function createAccount() {
         .catch(handleAPIError);
 }
 
+function backupAccount() {
+    const nickname = document.getElementById("nicknameCreate").value;
+
+    axios
+        .get(addPrefixUrl(`api/accounts/${nickname}/backup`))
+        .then((resp) => {
+            tableInsert(resp.data);
+            wallets.push(resp.data);
+        })
+        .catch(handleAPIError);
+}
+
```

Comment:
Is there any reason to not use our useResource here!? 🤔

Reply:
yes, it's legacy vanilla js, he updated this code to be able to test it quickly

Reply:
...but this could be using useResource, no!?

Reply:
no, it's not react, and we will remove this front as soon as the new one cover all its features

Reply:
Ahh ok! Got it!